forge/tests/it/config.rs

```diff
@@ -265,3 +269,15 @@ pub fn assert_multiple(
         }
     }
 }
+
+/// Converts an `EvmVersion` into a `SpecId`
+pub fn evm_spec(evm: &EvmVersion) -> SpecId {
+    match evm {
+        EvmVersion::Istanbul => SpecId::ISTANBUL,
+        EvmVersion::Berlin => SpecId::BERLIN,
+        EvmVersion::London => SpecId::LONDON,
+        EvmVersion::Paris => SpecId::MERGE,
+        EvmVersion::Shanghai => SpecId::SHANGHAI,
+        _ => panic!("Unsupported EVM version"),
+    }
+}
```

Comment:
This looks like it's the same as `foundry_cli::utils::evm_spec`, can we only use one?

Reply:
yep did this to quickly test the ethers bump and see if something broke. I think we could move this to `foundry_evm` and add that dependency to the CLI. wdyt?